neuralmonkey/runners/regression_runner.py

```diff
@@ -0,0 +1,79 @@
+# pylint: disable=unused-import
+from typing import Dict, List, Optional, Callable, Any
+# pylint: enable=unused-import
+
+import numpy as np
+import tensorflow as tf
+
+from neuralmonkey.model.model_part import ModelPart
+from neuralmonkey.runners.base_runner import (BaseRunner, Executable,
+                                              ExecutionResult, NextExecute)
+# pylint: disable=too-few-public-methods
+
+
+class RegressionRunner(BaseRunner):
+
+    def __init__(self,
+                 output_series: str,
+                 decoder: Any,
```

Comment:
Decoder v tomhle případě musí být ten `SequenceRegresson`, nebo ne? I kdyby ne, tak by to měl být aspoň `ModelPart`.